helmcharts/services/system-rules-ingestor/configs/rules.yaml

```
@@ -1,683 +1,377 @@
+  interval: 5m
+  labels: 
+    alert_code: "INFRA_KAFKA_001"
+  description: "The system is working slowly, which may delay the addition of new data."
+  annotations:
+    summary: "High CPU usage in Kafka may disrupt the data ingestion process, causing delays in data being processed and affecting the real-time flow of data into the dataset."
+  severity: warning
 
-- name: "Velero (Kubernetes Cluster) Backup Failure"
-  query: (time() - s3_last_modified_object_date{job="s3-backups", bucket=~"velero.*"})
-  operator: gt
-  threshold: [86400]
-  category: db backup
-  frequency: 1h
-  interval: 1h
-  labels: {}
-  description: The Kubernetes cluster backup has not occurred within the expected timeframe. Please Investigate immediately or contact administrative support for assistance.
-  annotations: {}
-  severity: critical
+- name: "[VALKEY]: High CPU Usage Detected. System Under Heavy Load."
+  query: sum by()(max by (label_system_infra,pod) (rate(container_cpu_usage_seconds_total{pod="valkey-dedup-primary-0"}[$__range]) / on (pod) group_left max by (pod) (kube_pod_container_resource_limits{resource="cpu"})) ) + sum by() (max by (label_system_infra,pod) (rate(container_cpu_usage_seconds_total{pod="valkey-denorm-primary-0"}[$__range]) / on (pod) group_left max by (pod) (kube_pod_container_resource_limits{resource="cpu"}))) * 100
+  operator: gt
+  threshold: [90]
+  category: "Infra"
+  frequency: 5m
+  interval: 5m
+  labels: 
+    alert_code: "INFRA_VALKEY_002"
+  description: "The system is working slowly, which may put datasets at risk of becoming unhealthy."
+  annotations:
+    summary: "High CPU usage in Valkey may stop the caching of new data, causing delays in processing real-time data and preventing new data from being queried."
+  severity: warning
 
-- name: "Kubernetes Node Storage Capacity Exceeded"
-  query: min by (instance) (max by (instance, mountpoint) (max_over_time(node_filesystem_avail_bytes{mountpoint!~".*/tmp.*"}[$__range]) > 0) / on (instance, mountpoint) max by (instance, mountpoint) (node_filesystem_size_bytes{mountpoint!~".*/tmp.*"}) * 100)
-  operator: lt
-  threshold: [20]
-  category: disk space
-  frequency: 2m
-  interval: 1m
-  labels: {}
-  description: The disk space on the Kubernetes node(s) has exceeded 80% of the total capacity. Please investigate immediately or contact administrative support for assistance.
-  annotations: {}
-  severity: critical
+- name: "[DRUID HISTORICAL]: High CPU Usage Detected. System Under Heavy Load."
+  query: max without(label_system_infra, pod)(max by (pod) (rate(container_cpu_usage_seconds_total{container='druid-raw-historicals'}[$__range]) / on (pod) group_left max by (pod) (kube_pod_container_resource_limits{resource="cpu"}))) * 100
+  operator: gt
+  threshold: [90]
+  category: "Infra"
+  frequency: 5m
+  interval: 5m
+  labels: 
+    alert_code: "INFRA_DRUID_HISTORICAL_003"
+  description: "The system is working slowly, which may prevent access to old data."
```

```suggestion
  description: "The system is working slowly, which may prevent access to old data."
```
It's unclear description.